docs/current_docs/manuals/developer/snippets/entrypoint/go/default-object/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "context"
+
+func New(
+	// +default=dag.Container().From("alpine:3.14.0")
```

Comment:
Error
```
Error: generate code: template: alias.go.tmpl:76:3: executing "_dagger.gen.go/alias.go.tmpl" at <ModuleMainSrc>: error calling ModuleMainSrc: failed to generate type def code for MyModule: failed to convert constructor to function def: default value "dag.Container().From(\"alpine:3.14.0\")" must be valid JSON: invalid character 'd' looking for beginning of value
```

Reply:
I don't think this is possible? In go you can just have it as `+optional` and then in the constructor say
```go
if ctr == nil {
    ctr = dag.Container().From("alpine:3.14.0")
}
```

Reply:
Updated and ready for review